File: app/src/main/java/in/completecourse/questionbank/SignupActivity.java
package in.completecourse.questionbank;

import android.os.Bundle;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import in.completecourse.questionbank.fragment.EasyLoginFragment;
import in.completecourse.questionbank.helper.HelperMethods;

public class SignupActivity extends AppCompatActivity {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_signup);

        //Load EasyLoginFragment
        HelperMethods.INSTANCE.loadFragment(new EasyLoginFragment(), this);

    }
}

File: app/src/main/java/in/completecourse/questionbank/SignupActivity.kt
package `in`.completecourse.questionbank

import `in`.completecourse.questionbank.fragment.EasyLoginFragment
import `in`.completecourse.questionbank.helper.HelperMethods
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity

class SignupActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_signup)

        //Load EasyLoginFragment
        HelperMethods.loadFragment(EasyLoginFragment(), this)
    }
}